wp-content/themes/goonj-crm/functions.php

```diff
@@ -95,8 +95,36 @@ function goonj_custom_password_reset_redirection( $errors, $user ) {
 	}
 }
 
-add_shortcode( 'goonj_check_user_form', 'goonj_check_user_action' );
+add_shortcode('goonj_check_user_form', 'goonj_check_user_action');
```

Comment:
Can you do it in this way only?

```suggestion
add_shortcode( 'goonj_check_user_form', 'goonj_check_user_action' );
```